vue-app/src/views/Landing.vue

```diff
@@ -515,32 +508,21 @@ ol li::before {
   grid-template-columns: 3fr 2fr;
   grid-template-rows: repeat(2, auto);
   grid-template-areas: 'image content' 'image .';
-  @media (max-width: $breakpoint-m) {
+  @media (max-width: $breakpoint-l) {
```

Comment:
I've changed the breakpoint because the "how it works" content was shrinking too much before hitting the md size.